.pylintrc

```diff
@@ -0,0 +1,407 @@
+[MASTER]
```

Comment:
This seems to be the pylint configuration file. I'm not familiar with pylint. Could you please describe why this file is needed in the repo?

Reply:
That's quite optional. Pylint is a static code analyzer. It polishes the code format and is able to find some mistakes. I just use it in all the python projects to check the files before pushing to public repos. I think it will be useful here too.

Reply:
I think I use something like this, integrated into vim (it reports about syntax errors, formatting incompatible with pep8 etc.). I tried ``pylint main.py`` and it generated a huge list of messages: the resulting table reported about 375 invalid names. Do you use some command line parameters to control the output?

Reply:
There should be some options, you can add pylint ignore statements as special comments in the code. I've never met that many invalid names. I'm just forced to follow the naming rules from the very beginning. There are some exclusions In pylintrc, you can add their yours.